test/documentation/seqan3.css

```diff
@@ -131,3 +151,54 @@ table.doxtable th {
Ex9uHjzvecnZ3pB+7cW/tsn6nBqOpII91vZm92fv/3zAL/b1PbtPEeeP0VffPDdfhgTQMpaT4kmm83Hm9/6ngPpJJ8w0/WP2zaKf7SVOCbIvDGG2/UNjU1fbN582YppdT8hR4IKbXbjbds2SKampoub9q0af6ECJim+dqDD9bcVbdsqWt86dIlLrxkySIXXrS41oVrFy1w4QULa1y45kH3vubX3O/CVdWVyuw5FbMsy/r1hAgoinKPTnV0dV2Aoqq4HopBVTWcbm9z4bbTp1z4VFvrf7CiofXkCRc+0XrchY+fOOrCR4996cKdnR3QiQ4AFRNy4rVr1/5yxswZBcFgPyCi+OLAZ/CWrgLx3Q8NCXxx8AA801dC81ZDVRL48uABmCUPQfNUQ1MSOHrwAIxpD0H1VN3Ahw6CBhqgWJXQlASOHT4EvfjbgDlvCJOiekjjPhAliWNHDkPzLweseSgrLcfAQCi1a9eu13MKO9fg1q1bB5cuXeJr7zgNIQSklEMdgAtLSEBi6LeU4+OhsXGwRjTU1i5GZ8fZ0IYNGwK59kqGD0gplbfeesuqqJiNhQsXIcf8qBHhVswlkwl0nD7jy9sHtm3b5qeUOsXFxW5Jj6aBWzzn8XihaZq6fft2My8NMMYCRUVFtpSS3m5p55qTUsIwDBaNRgMAesYlIIQImKbpZC/Y3Nw84RR/syQy7ZFHHhl61rIsHolE8iOgaVrA4/G4XrZ69erbrols07IsS2qalrcTByzLUrMXEULcEmnnowkpJUzTVKSU+RFQFCXg8XhINoG9e/fm9bLJJNHY2JjlyB4tbwJSyoBhGDSbQGNjY14Sm2znzdKAHuk/X7luVcEriiKKhdA++MP+SEtOAoSQUsMwXCY0EbVPtjn1XD6D3e/8So9cufjzDU88oxnJBP50aPdLP2pUNv5+b3jbCAKqqs4wDMPlSHv27JkwiUwUuVkSX104iY/efRXh3q/wg4Y1qFr4JFG7e4HublTp1dZLOPzb59binVwEplNKkcuEbocmzrd/jn+8/xrseARr1jwPXvgtsFAIzu7dULu6gJ4eiFAIcyo12RkvWJArD5RkCPy3UWQi/z176hA+evdVxCK9WFlXiXn3rQIciUtCYLCgALSwEOVtbejnHH0OxzfghNtKkORYu4gQMiIW3woSQgi0H9uPj997FdKOYcWyStx771IoUCFFHMnwX1HQVY++2TX4uqgIPfE4BiyBi1VKKqXLw+/tHzw3goDjOP5cBCaThJQCJ1ua8fGO1+A1FKyufwB33zUTCgDB44gHO5EKfw0pHcSvT4N9dxWil86h5wFpnzbBBJE7wKIvjhZGvYQQcM6Hxvbt2zcpSUk4NrraPkFX698xZ1Y5nmhcgLLSshuCY4NIXOtEKtINSAEpgd4QR8+1MOQnf8SJ7s8QN8VvuFA2vb97MJjzPPDmm2/6NE0bWLdunc4YmzTntFkSh5rfwScfbkH13FmoWzgXgcA0AABPhpAIngGLXgUg4QiJy302zl9huBanYJxg9ZM/w6WgIgzTY65fv94eNZFxzgMej8cGoE9GPE/EB/HPXb/D4d1vY37lbDzzVCOKi4pvkIr3IR48Ax6/dqMK5hKXr9q40MvQH6MgRgCN39uI2vq18Pn82LlzJ7NtuxhA36gECCEB0zT5cOfNmNBESHSfPYzOz/+Mmuq5ePbpNSjy+29sNHoFiYFzQxtPMIlLvQyX+hmuXFegUi/m1D6Gp599GZpGsrMxTyQSgTEJCCEClmWNcOCVK1dOSBNH9r6P7tN/w0+f/z58BIAEWKwXiWAneGIAABBNCly8wvBVP8fVQRUl5ffiqRc2orp2RaYmcxWTpmkKIURgzFoolUoFKKVq5hw8Xst+SaYd+Hg7/tXchBd+sRFWYBaSV07h+sm/wEmFAQChqIOz3QxdV22EYgLzF6/C44+/iJkV1aMKJ60B5CqpXQRs2w5QSsnwsLl///686veuU3vRd24Pnnt+HQgh4LF+qN4AAAd91/mNjfdxhBMO7pm7DNULHoWvsBSdF3vRebHXteaKFStcVkAI0VKpVF4E9OEEGhoackqFMQbbtmHbNlr2vYdksA2Prv0udCLBo/2AqgGS44uzCbSejyPFgPnLH8P8ZU+gsLgUhBDoug5CCCiloJRCVdUR1agQApRSnXM+NgEhRCmlVBtuQoqiuExGCAHGGFKpFGzbRuuRD3H8sx040NqPlpOX8cNHZyMc4xgIJRAMXUcslsQ9VSswb/F34PNPg4COWCwGSikcx0F27UUpRSaRZguLUqozxsYlUE4pHSHtTz/9dITJOI4Dxhi6zx3BhaMfwFdQgDUNlfAWzkRQnQGzvASz5gZQWVACwyqAoihQVXVI4oQQaJoGTdOgqqqrj8zcEpRSBUD5mAQcxymllA4dITMLLV++fNQIFK6tgfXjl8E5H+qO48BxHGRrUlGUIRKZjWeIZKQ+WnRLmxA452Xj+cC04WcBKSU0TRvVeQPTZ+QcF0KMOEtnSGSbZb6nM13XwTkvGZNAMpks1nUdw33gZrKyqqqjbnKia2Y0wBibNl4e8BuGgVx5YCovuYQQ0HUdyWSycEwCiUTCp+s6GGMTPg/fyjkhBAzDQDKZLBj1brSurs7POVcJISNupKfqXjTbn3RdRyqVssrKyry5NOCNx+MVhBBmGIaVfTN3pzTbtiGl5F6v9y4AvQBiABwCwALgsW07oGmaaG9vR0dHxx1HoKqqCqqqZgq6CAABIEoA6AAoY0xnjCmO40DX9TuOAOccUkpwzg0ANL1vki52gXA4HB4YGCC2baOiouKONKFgMKhFIpH48E9MJgAfAF99ff16v9//XD4fwG93k1LyUCi0taWl5W0A0XSPZTKNJ92tNCGa5eBkKvcNwEl3G0ASQAJAPO3EUhkWkcws+9LSGppqbfC0w3IALE2CjfmVMp0ftHRXp1gDGQIO/hfbvwF6ZBhg2gmmVgAAAABJRU5ErkJggg==);
```

Comment:
?

Reply:
this adds the paper & pencil icon from seqan2 to the assignment header 😁 💅

Reply:
Ok, but why this wired hash?

Reply:
How do I know :sweat_smile: I literally have no CSS experience. I just "inspect the page" in my browser and then copy and paste what I find... (When you inspect the icon in seqan2, this is hash is what I get. And it works :grin: If you tell me how I can inlcude an image in css AND how doxygen may find it with his autogenerated html pages I'll gladly change it to some proper file location.)

Reply:
I suspect that the url will become invalid if we take down the seqan2 readTheDocs page but we are not planning to are we?

Reply:
The string is the encoded image and no url. You can paste it to a png base 64 converter and you'll get the image.

Reply:
Ok, it also seems to be a good practice for css in order to save http requests.